### ISSUE TITLE
Fx Lab examples code style is extremely difficult to understand and reuse.

### ISSUE THREAD

**Issue by renetik** [google/oboe#1649]
I developed huge android app for music production and used bunch of c++ files across the available github repositories but I am struggling to understand style of FXLAB app sample. 

I believe if this is en example of how to do things right this should have som documentation explaining how it works even for experienced developers like me. Maybe I am not hardcore C++ developer (that can be issue here) but I can reuse most of code I read and find on internet, not this one sadly, after multiple tries I cannot incorporate anything from those FX examples into my app.

Thanks.

**Comment by philburk** [comment]
I agree. We need some better README files for the FXLab sample that describe how to apply this code.

**Comment by Gilianp** [comment]
I also can't reuse the c++ code from the fxlab project. It seems very confusing to me. Unfortunately, one of the most interesting projects becomes almost useless. The code style is very complex
